docs/userguide/constraints.md

```diff
@@ -1,4 +1,259 @@
 # Constraints
+Experimental campaigns often have naturally arising constraints on the parameters and 
+their combinations. Such constraints could for example be:
+* When optimizing a mixture, the relative concentrations of the used ingredients must 
+  add up to 1.0.
+* For chemical reactions, a reagent might be incompatible with high temperatures, hence 
+  these combinations must be excluded.
+* Certain settings are dependent on other parameters, e.g. a set of parameters only 
+  becomes relevant if another parameter called "Switch" has the value "on".
 
-This page will soon contain information about constraints.
-In the meantime, please see the [examples](../../examples/examples) instead.
\ No newline at end of file
+Similar to parameters, BayBE distinguishes two families of constraints: discrete and 
+continuous constraints. 
+A constraint is called discrete / continuous if it operates on a set of exclusively 
+discrete / continuous parameters.
+
+```{note}
+Currently, BayBE does not support hybrid constraints, that is, constraints which 
+operate on a mixed set of discrete and continuous parameters. If such a constraint is 
+necessary in almost al cases it is possible to rephrase the parametrization so that 
+the parameter set is exclusively discrete or continuous.
+```
+
+## Continuous Constraints
+
+```{warning}
+Not all surrogate models are able to treat continuous constraints. In such situations 
+the constraints are currently silently ignored.
+```   
+
+### ```ContinuousLinearEqualityConstraint```
```

Comment:
By now, double backticks should be sufficient since the triple one was just necessary for the markdown conversion (only marking here, I guess that this happens in several places).

Reply:
Not sure about the "envorinments" like python. I think they still need triple backticks.

Reply:
In the code, we used double, so using double everywhere else but in the environments would be the more consistent approach.

Reply:
done